WeakAuras/WeakAuras.lua

```diff
@@ -6432,12 +6432,12 @@ local function GetAnchorFrame(region, anchorFrameType, parent, anchorFrameFrame)
 
   if (anchorFrameType == "CUSTOM" and region.customAnchorFunc) then
     WeakAuras.StartProfileSystem("custom region anchor")
```

Comment:
I was expecting something like this, honestly.
```suggestion
	if not WeakAuras.IsLoginFinished() then
		return parent
	else
	    WeakAuras.StartProfileSystem("custom region anchor")
	    -- etc
```

Reply:
i want to implement this, but i'm leveling my priest :)